CODEREVIEW

```diff
@@ -0,0 +1,5 @@
+// @CODEREVIEW: General remarks:
+// @CODEREVIEW:
+// @CODEREVIEW: Remove all unused code (code in comments)
+// @CODEREVIEW: Please run `phpcs` and get rid of all coding standard errors and warnings.
+// @CODEREVIEW: Consider using `===` for all comparisons (rather than just `==`).
```

Comment:
almost all places are changed from == to ===

Reply:
Cool.